GameProject/Assets/TurnBasedAssets/Scripts/RotateCube.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+
+namespace TurnBasedAssets.Scripts
+{
+    public class RotateCube : MonoBehaviour, ISelection
+    {
+        public void Select()
+        {
```

Comment:
You could convert these methods in here to expression-bodied methods.

Reply:
These methods are from the interface, the functionality will change depending on what they are used for. I'm not sure expression-bodied methods would be useful here.

Reply:
Expression bodied methods change nothing about the interface signature or not - its just syntax sugar. Example:
```
        public void Select() => MoveCube();
        
```

Reply:
Oh I get it now! Thanks!